testing/locally/local-testing.sh

```diff
@@ -19,32 +19,45 @@ rm -f tmp/kube-audit-rest.log;
 echo "Removing old servers if still running"
 pkill kube-audit-rest || echo "No old server running"
 
+if which python3
```

Comment:
Let's not do this, the sockets we just tried to bind to may not be free and it's better to not require yet another thing (Python). Let's just use a specific port that  people can override if needed

Reply:
I did that because the port 55555 was not free in my laptop and, because of that, the test failed.

Python is not a required dependency. If python is not installed, this `if` statement will be false and then we'll choose `SERVER_PORT=9090` and `METRICS_PORT=55555` instead of binding to a random free port.

If you don't like the python stuff, then we can allow the user to select the ports using command line arguments. Something like `testing/locally/local-testing.sh --server-port <port> --metrics-port <port>`. If these arguments are not specified, then we'll use the default ports (9090 and 55555).

What do you think?

Reply:
We'll keep it as it is, since this if statement doesn't force the user to install python.